storage/FileManagerPosix.cpp

```diff
@@ -86,7 +88,7 @@ size_t FileManagerPosix::numSlots(const block_id block) const {
   }
 
   if ((file_stat.st_size % kSlotSizeBytes) != 0) {
-    throw CorruptPersistentStorage();
+    LOG(FATAL) << "The file " << filename << " was corrupted.";
```

Comment:
If you change this file, change other two `FileManager` implementations as well.